src/bobbit/message.py

```diff
@@ -13,12 +13,21 @@ def __init__(self, body, nick=None, channel=None, notice=False, highlighted=Fals
         self.timestamp   = timestamp or time.time()
 
     def copy(self, body=None, nick=None, channel=None, notice=None, highlighted=None, timestamp=None):
+        notice_truthy = notice
```

Comment:
I like the idea of the checks against `None`, but let's just call these `notice` and `highlighted`, rather than adding the `_truthy` suffixes.

Reply:
Removed separate variable, so that boolean variable values are overwritten